src/qtiCommonRenderer/helpers/Graphic.js

```diff
@@ -226,20 +227,35 @@ var GraphicHelper = {
          * @private
          */
         function resizePaper(e, givenWidth) {
-            var maxWidth, containerWidth, containerHeight, factor;
+            var diff, maxWidth, containerWidth, containerHeight, factor;
```

Comment:
**quibble:** It would have been nice to also convert this part to ES6. Anyway, this is not a blocker.